loops-and-arrays.js

```diff
@@ -41,6 +51,9 @@ const numbers = [2, 4, 5, 29, 38];
 // ==========================================
 
 const squares = [30, 2, 8, 24, 11];
+for ( let  i = 0; i < numbers.length; i++ ){
+    console.log("Het volume van",+ squares[i] +" is "+ squares[i] ** 3);
+}
 
```

Comment:
Je code werkt goed en je gebruikt ****3**, daarmee schrijf je de code zelfs korter dan in de uitwerkingen van je word verwacht. Goed gedaan! Alleen word er in de opdracht wel gevraagd om de orginele waardes in de array te overschrijven. Met deze code log je wel de juiste tekst maar pas je niet de waardes in de array aan. Misschien kan je hier nog even naar kijken.